config/routes.rb

```diff
@@ -15,7 +16,7 @@
 
   resources :videos, only: [:show, :index] do
     collection do
-      get :search, to: "videos#search"
+      get :search, to: 'videos#search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.